src/app/model/tollbooth-factory.ts

```diff
@@ -42,5 +42,13 @@ export class TollboothFactory {
       tollbooth.beginGeneration();
       tollbooths.push(tollbooth);
     }
+
+    this.renderTargets(lanes, top);
+  }
+
+  renderTargets(lanes: number[], offset: number): void {
+    lanes.forEach(lane => this.world.ref.append("rect")
```

Comment:
Somehow all these refs should also be accessible in the world, maybe just returned from here rather than packaged with the positions of the targets somehow?